frontend/Dockerfile.prod

```diff
@@ -1,4 +1,4 @@
-FROM node:10.16.0-alpine
+FROM node:14.16.1-alpine3.13
```

Comment:
Something I realized recently: we need to use less specific version (e.g. `node:14`) or we will never pull in security updates.

Reply:
Resolved here 1d6a27b.